shm/include/coordinates.hpp

```diff
@@ -1,9 +1,11 @@
 #pragma once
+#include <cstddef>
```

Comment:
What for is this header?

Reply:
I had compilation errors that size_t type is unknown. That's why I added this header, however I also find this quite strange.. I thought that size_t doesn't need any additional include. But perhaps it was because other libraries that we used included size_t type definition?